tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+project(SOF_FUZZER C)
+
+include(ExternalProject)
+
+set(source_directory "${PROJECT_SOURCE_DIR}/../testbench")
+
+ExternalProject_Add(sof_tb
+	SOURCE_DIR "${source_directory}"
+	PREFIX "${PROJECT_BINARY_DIR}/sof_tb"
+	BINARY_DIR "${PROJECT_BINARY_DIR}/sof_tb/build"
+	CMAKE_ARGS -DBUILD_LIBRARY=ON
+		-DCMAKE_INSTALL_PREFIX=${PROJECT_BINARY_DIR}/sof_tb/build
+		-DCMAKE_VERBOSE_MAKEFILE=${CMAKE_VERBOSE_MAKEFILE}
+	BUILD_COMMAND make
```

Comment:
Command shouldn't be hardcoded to make for 2 reasons.
Someone can use f.e. ninja, it's better to leave it to cmake to use proper command. (If you specify no command, then it's something like `make all`, but for correct buildsystem). Also in case if it is not root project it will break recursive make (splitting jobs from -j flag etc).

Reply:
Also you shouldn't build whole testbench just for lib. Build just target that makes .so.

Reply:
> Also you shouldn't build whole testbench just for lib. Build just target that makes lib.

@jajanusz i couldnt find any examples of building just one targer in the external project. How do I do this?

Reply:
Something like `BUILD_COMMAND ${CMAKE_COMMAND} --build . --target tplg_parser`. Anyway if you decide to switch to one buildsystem for all binaries in tools/ folder to make things easier (and still managable/decopuled) you have a green light from me ;p

Reply:
@jajanusz thanks. Yes, I think it makes sense to consolidate the buildsystem. Let me make an attempt today